### ISSUE TITLE
MediaManager.Dispose: InvalidOperationException: Collection was modified; enumeration operation may not execute.

### ISSUE THREAD

**Issue by Dummiesman** [DubyaDude/WindowsMediaController#17]
I'm experiencing this error both in my own application, and the provided sample applications
It seems that .Dispose() on sessions modifies the keys being iterated in MediaManager.Dispose

```
Unhandled Exception: System.InvalidOperationException: Collection was modified; enumeration operation may not execute.
   at System.ThrowHelper.ThrowInvalidOperationException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.KeyCollection.Enumerator.MoveNext()
   at WindowsMediaController.MediaManager.Dispose() in /_/WindowsMediaController/Main.cs:line 253
   at Sample.CMD.Program.Main() in D:\a\WindowsMediaController\WindowsMediaController\Sample.CMD\Program.cs:line 32
```

**Comment by DubyaDude** [comment]
Was able to reproduce it, looking into it now.

**Comment by DubyaDude** [comment]
Resolved in version [2.5.5](https://github.com/DubyaDude/WindowsMediaController/releases/tag/2.5.5)
May take a few minutes for Nuget to index the latest version.

Thanks for reporting, and please let me know if you have any other issues
